changelog.d/16010.misc

```diff
@@ -0,0 +1 @@
+Update dehydrated devices implementation.
```

Comment:
Let's use the same as #15929 since that hasn't landed yet:

```suggestion
Implement [MSC3814](https://github.com/matrix-org/matrix-spec-proposals/pull/3814), dehydrated devices v2/shrivelled sessions and move [MSC2697](https://github.com/matrix-org/matrix-spec-proposals/pull/2697) behind a config flag. Contributed by Nico from Famedly and H-Shay.
```

(Also needs to be renamed to feature.)

Reply:
I am slightly confused, #15929 was released already as part of v1.89.0 - happy to change the changelog but I am confused about it not being landed?

Reply:
Huh, the GitHub UI isn't showing it as part of that tag... My mistake!